back/config/routes.rb

```diff
@@ -95,6 +95,8 @@
 
         resources :comments, only: [:index], controller: 'user_comments'
       end
+
+      get 'users/:attendee_id/events', to: 'events#index_by_attendee'
```

Comment:
I think I would have nested this under `/events/by_attendee` or something instead, just to have a cleaner colocation between routes and implementation, and more in line with what we've typically done. Not a blocker.

Reply:
I don't quite understand what you mean by "cleaner collocation between routes and implementation."

- The use of `by_attendee` seems redundant with `users` (unless we foresee adding an owner to events in the future, but I think that's too much anticipation for an internal API).
- I slightly prefer the name `index_by_attendee` over `by_attendee` for the action name.
- It's not very RESTful ; not that it really matters. However, I usually avoid putting a sub-route where you would expect an identifier when possible.